packages/cadl-ranch-specs/http/special-headers/repeatability/main.tsp

```diff
@@ -0,0 +1,28 @@
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+@doc("Illustrates OASIS repeatability headers")
+@supportedBy("dpg")
+@scenarioService("/special-headers/repeatability")
+namespace SpecialHeaders.Repeatability;
+
+@scenario
+@scenarioDoc("""
+Check we recognize Repeatability-Request-ID and Repeatability-First-Sent.
+""")
+@doc("""
+Check we recognize Repeatability-Request-ID and Repeatability-First-Sent.
+""")
+@post
+@route("/doSomething")
+op doSomething(
```

Comment:
nit, do we have better name than `doSomething`? Maybe just `action` or `post`?

Reply:
I think depending on all teh scenarios we need (maybe different verb from @tadelesh) comment and different retry flow this would be better to be named describing what it is checking.
Base scenario would probably just check that you are passing the headers and it would succeed. So maybe 
```
immediateSuccess
```